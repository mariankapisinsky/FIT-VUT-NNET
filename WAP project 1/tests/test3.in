53,658,566,362,749,888,477,289,592,662,493,271,493,457,511,516,433,382,818,84,876,691,881,148,785,583,508,619,90,311,34,504,991,673,506,291,831,487,703,918,240,697,881,861,780,75,39,714,286,756