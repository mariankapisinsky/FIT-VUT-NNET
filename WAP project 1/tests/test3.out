preorder
53,34,39,658,566,362,289,271,84,75,148,90,240,286,311,291,477,457,433,382,493,487,493,511,508,504,506,516,592,583,619,749,662,691,673,703,697,714,888,818,785,780,756,876,831,861,881,881,991,918
inorder
34,39,53,75,84,90,148,240,271,286,289,291,311,362,382,433,457,477,487,493,493,504,506,508,511,516,566,583,592,619,658,662,673,691,697,703,714,749,756,780,785,818,831,861,876,881,881,888,918,991
postorder
39,34,75,90,240,148,84,286,271,291,311,289,382,433,457,487,506,504,508,516,511,493,493,477,362,583,619,592,566,673,697,714,703,691,662,756,780,785,861,831,881,881,876,818,918,991,888,749,658,53